libs/entitlement/src/test/java/org/elasticsearch/entitlement/runtime/policy/PolicyParserTests.java

```diff
@@ -182,10 +182,7 @@ public void testParseFiles() throws IOException {
                       mode: "read"
                     - path: '%s'
                       mode: "read_write"
-                    - path_setting: foo.bar
-                      mode: read
-                    - relative_path_setting: foo.bar
-                      relative_to: config
+                    - config_path_setting: foo.bar
```

Comment:
Hmm. This is the first entitlement whose meaning is not immediately obvious to me. It could be particularly confusing if the path has nothing to do with the `config` dir (ie. if it's an absolute path). `path_setting` made sense to me, because it's a setting representing a path.

Anyway I don't feel strongly about it. If you think plugin authors will understand this, that's what matters. And assuming we want all three words "config", "path", and "setting" in the name, I think you've picked the best possible order. 😂

Reply:
Yeah, I find it a bit awkward too. I also tend to think `path_setting` makes sense but it doesn't convey the implicit handling paths relative to config. But, to be honest, I'm not sure prefixing `config` to the name makes things much clearer.

This also keeps the door open to introducing other "path setting" entitlements in the future that have slightly different semantics.

Reply:
I'm not sure there's a need to convey that relative paths relate to `config` in the name of this field. That could be just how `path_setting` works, and mentioned in documentation. The first time someone says "huh, a relative path; I wonder what it's relative to?" the first place they'd look would be the docs for `path_setting`.